gui/src/i18n/de/translation.json

```
@@ -0,0 +1,462 @@
+            "chest": "Brust",
+            "waist": "Taille",
+            "knees": "Knie",
+            "feet": "Füße",
+            "elbows": "Ellbogen"
+          }
+        }
+      }
+    }
+  },
+  "onboarding": {
+    "skip": "Einrichtung überspringen",
+    "continue": "Fortsetzen",
+    "wip": "Noch in Bearbeitung",
+    "wifi-creds": {
+      "back": "Zurück zur Einführung",
+      "title": "WLAN-Zugangsdaten eingeben",
+      "description.p0": "Die Tracker nutzen diese Zugangsdaten, um sich mit dem WLAN zu verbinden",
+      "description.p1": "Bitte verwenden Sie die Zugangsdaten, mit denen ihr PC gerade verbunden sind",
+      "skip": "WLAN-Zugangsdaten überspringen",
+      "submit": "Weiter!"
+    },
+    "reset-tutorial": {
+      "back": "Zurück zur Montagekalibrierung",
+      "title": "Tutorial zurücksetzen",
+      "description": "Diese Funktion ist noch nicht fertig, drücken Sie einfach auf Fortsetzen"
+    },
+    "home": {
+      "title": "Willkommen zu SlimeVR",
+      "description.p0": "VR-Fullbody Tracking",
+      "description.p1": "für jeden",
+      "start": "Lass uns loslegen!"
+    },
+    "enter-vr": {
+      "back": "Zurück zur Trackerzuweisung",
+      "title": "Zeit für VR!",
+      "description": "Ziehen Sie alle Tracker an und betreten Sie dann VR!",
+      "ready": "Ich bin bereit"
+    },
+    "done": {
+      "title": "Alles eingerichtet!",
+      "description": "Genießen Sie Ihre Fullbody-Erfahrung",
+      "close": "Schließen Sie die Einrichtung"
+    },
+    "connect-tracker": {
+      "back": "Zurück zu WLAN-Zugangsdaten",
+      "title": "Verbinde Tracker",
+      "description.p0": "Nun zum unterhaltsamen Teil, verbinde alle Tracker!",
+      "description.p1": "Verbinden Sie einfach alle Tracker, die noch nicht verbunden sind, über einen USB-Anschluss.",
+      "issue": {
+        "serial": "Ich habe Schwierigkeiten die Tracker verbinden!"
+      },
+      "usb": "USB Tracker",
+      "connection-status": {
+        "connecting": "Sende WLAN-Zugangsdaten",
+        "connected": "Mit WLAN verbunden",
+        "error": "Kann nicht mit WLAN verbinden",
+        "start-connecting": "Suche nach Trackern",
+        "handshake": "Mit dem Server verbunden"
+      },
+      "connected-trackers": "{{amount}} Tracker verbunden",
+      "next": "Ich habe alle meine Tracker verbunden."
+    },
+    "assign-trackers": {
+      "back": "Zurück zu den WLAN-Anmeldedaten",
+      "title": "Tracker zuweisen",
+      "description": "Wählen sie nun aus, welcher Tracker wo hin geht. Klicken Sie auf einen Ort, an dem Sie einen Tracker platzieren möchten",
+      "assigned": "{{assigned}} von {{trackers}} Trackern zugewiesen",
+      "advanced": "Erweiterte Zuweisungspositionen anzeigen",
+      "next": "Ich habe alle Tracker zugewiesen"
+    },
+    "manual-mounting": {
+      "back": "Zurück zum Eintritt in die VR",
+      "title": "Manuelles Befestigen",
```

Not sire if too long. Suggestions welcome
```suggestion
      "title": "Manuelle definiton der Befestigungsposition",
```

---

In which window was this? Didn't find it

---

Updated it, let's just hope that it works out